src/transformers/models/auto/image_processing_auto.py

```diff
@@ -37,104 +45,125 @@
+    IMAGE_PROCESSOR_MAPPING_NAMES: OrderedDict[str, Tuple[Optional[str], Optional[str]]] = OrderedDict()
+else:
+    IMAGE_PROCESSOR_MAPPING_NAMES = OrderedDict(
+        [
+            ("align", ("EfficientNetImageProcessor",)),
+            ("beit", ("BeitImageProcessor",)),
+            ("bit", ("BitImageProcessor",)),
+            ("blip", ("BlipImageProcessor",)),
+            ("blip-2", ("BlipImageProcessor",)),
+            ("bridgetower", ("BridgeTowerImageProcessor",)),
+            ("chinese_clip", ("ChineseCLIPImageProcessor",)),
+            ("clip", ("CLIPImageProcessor",)),
+            ("clipseg", ("ViTImageProcessor", "ViTImageProcessorFast")),
+            ("conditional_detr", ("ConditionalDetrImageProcessor",)),
+            ("convnext", ("ConvNextImageProcessor",)),
+            ("convnextv2", ("ConvNextImageProcessor",)),
+            ("cvt", ("ConvNextImageProcessor",)),
+            ("data2vec-vision", ("BeitImageProcessor",)),
+            ("deformable_detr", ("DeformableDetrImageProcessor",)),
+            ("deit", ("DeiTImageProcessor",)),
+            ("depth_anything", ("DPTImageProcessor",)),
+            ("deta", ("DetaImageProcessor",)),
+            ("detr", ("DetrImageProcessor",)),
+            ("dinat", ("ViTImageProcessor", "ViTImageProcessorFast")),
+            ("dinov2", ("BitImageProcessor",)),
+            ("donut-swin", ("DonutImageProcessor",)),
+            ("dpt", ("DPTImageProcessor",)),
+            ("efficientformer", ("EfficientFormerImageProcessor",)),
+            ("efficientnet", ("EfficientNetImageProcessor",)),
+            ("flava", ("FlavaImageProcessor",)),
+            ("focalnet", ("BitImageProcessor",)),
+            ("fuyu", ("FuyuImageProcessor",)),
+            ("git", ("CLIPImageProcessor",)),
+            ("glpn", ("GLPNImageProcessor",)),
+            ("grounding-dino", ("GroundingDinoImageProcessor",)),
+            ("groupvit", ("CLIPImageProcessor",)),
+            ("idefics", ("IdeficsImageProcessor",)),
+            ("idefics2", ("Idefics2ImageProcessor",)),
+            ("imagegpt", ("ImageGPTImageProcessor",)),
+            ("instructblip", ("BlipImageProcessor",)),
+            ("kosmos-2", ("CLIPImageProcessor",)),
+            ("layoutlmv2", ("LayoutLMv2ImageProcessor",)),
+            ("layoutlmv3", ("LayoutLMv3ImageProcessor",)),
+            ("levit", ("LevitImageProcessor",)),
+            ("llava", ("CLIPImageProcessor",)),
+            ("llava_next", ("LlavaNextImageProcessor",)),
+            ("mask2former", ("Mask2FormerImageProcessor",)),
+            ("maskformer", ("MaskFormerImageProcessor",)),
+            ("mgp-str", ("ViTImageProcessor", "ViTImageProcessorFast")),
+            ("mobilenet_v1", ("MobileNetV1ImageProcessor",)),
+            ("mobilenet_v2", ("MobileNetV2ImageProcessor",)),
+            ("mobilevit", ("MobileViTImageProcessor",)),
+            ("mobilevit", ("MobileViTImageProcessor",)),
```

Comment:
@amyeroberts Is this typo or is there any specific reason for duplication?

Reply:
I have found this when I try to resolve conflict from local branch

Reply:
It's just a typo - you can update and remove one of the lines. Thanks for catching!

Reply:
I think it must have been added when mobilevit was first added, as it was present in the previous automap list. I've opened a PR here to remove it: https://github.com/huggingface/transformers/pull/31383